### PR TITLE
Creating Adaptivity for StreetCode catalog

### PR DESCRIPTION
**Pull request by ValDekh**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions
- [x]  Closed issue #188

### CODE REVIEW

File: package-lock.json
Comment (Tysyatsky) [inline package-lock.json:1]: remove this file and don't commit in the future.

**Review by Tysyatsky — APPROVED** [review]
Great First big issue.

**Review by MaksBrat — APPROVED** [review]

File: src/app/api/media/images.api.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import Agent from '@api/agent.api';
 import { API_ROUTES } from '@constants/api-routes.constants';
-import Image from '@models/media/image.model';
+import Image, { ImageCreate } from '@models/media/image.model';
 
 const ImagesApi = {
     getAll: () => Agent.get<Image[]>(`${API_ROUTES.IMAGES.GET_ALL}`),
 
-    getById: (id: number) => Agent.get<Image>(`${API_ROUTES.IMAGES.GET}/${id}`),
+    getById: (id: number) =>{ console.trace(); console.log(id); return Agent.get<Image>(`${API_ROUTES.IMAGES.GET}/${id}`)},
```
Comment (Kotusyk) [inline src/app/api/media/images.api.ts:8]: Does these console logs necessary here? If you use it only in the development stage - it need to be removed

File: src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx [hunk 2]
```diff
@@ -27,23 +29,25 @@ const NewStreetcode = () => {
...
                         <MainBlockAdmin form={form} />
                         <TextBlock />
                     </Form>
                     <InterestingFactsBlock />
-                    <ArtGalleryBlock />
-                    <RelatedFiguresBlock />
-                    <TimelineBlockAdmin />  
-                    <ForFansBlock />
+                    <ArtGalleryBlock arts={arts} setArts={setArts} />
+                    {/*  <RelatedFiguresBlock /> */}
```
Comment (Kotusyk) [inline src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx:45]: Does RelatedFiguresBlock should be commented?

File: src/features/AdminPage/NewStreetcode/MapBlock/MapAdmin/MapAdmin.component.tsx [hunk 3]
```diff
@@ -1,138 +1,149 @@
...
-  const [streetcodeCoordinates, setStreetcodeCoordinates] = useState<StreetcodeCoordinate[]>([]);
-  const mapRef = useRef<google.maps.Map | null>(null);
+    const [autocomplete, setAutocomplete] = useState<google.maps.places.Autocomplete | undefined>(undefined);
+    const [center, setCenter] = useState(initialCenter);
+    const [streetcodeCoordinates, setStreetcodeCoordinates] = useState<StreetcodeCoordinate[]>([]);
+    const mapRef = useRef<google.maps.Map | null>(null);
 
-  const handleSaveButtonClick = () => {
+    const handleSaveButtonClick = () => {
     // TODO: save streetcodeCoordinates to the backend
```
Comment (Kotusyk) [inline src/features/AdminPage/NewStreetcode/MapBlock/MapAdmin/MapAdmin.component.tsx:34]: Can you remove it, please?

File: src/features/AdminPage/NewStreetcode/TimelineBlock/NewTimelineModal/NewTimelineModal.component.tsx [hunk 4]
```diff
@@ -124,7 +124,7 @@ const NewTimelineModal:React.FC<{ timelineItem?:TimelineItem, open:boolean,
                                     value={cntx.title}
                                 />
                             ))}
-                    </Select>
+                    </Select> */}
```
Comment (Kotusyk) [inline src/features/AdminPage/NewStreetcode/TimelineBlock/NewTimelineModal/NewTimelineModal.component.tsx:127]: Need to be uncommented, but as far, as I know this is fixed

File: src/app/api/media/images.api.ts (hunk 1 above)
Comment (ValDekh) [reply src/app/api/media/images.api.ts:8]: These are not my decisions. It came from branch master when I'd merged it before

File: src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx (hunk 2 above)
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx:45]: These are not my decisions. It came from branch master when I'd merged it before

File: src/features/AdminPage/NewStreetcode/MapBlock/MapAdmin/MapAdmin.component.tsx (hunk 3 above)
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/MapBlock/MapAdmin/MapAdmin.component.tsx:34]: Done

File: src/features/AdminPage/NewStreetcode/TimelineBlock/NewTimelineModal/NewTimelineModal.component.tsx (hunk 4 above)
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/TimelineBlock/NewTimelineModal/NewTimelineModal.component.tsx:127]: It's not my decisions. It came from branch master when I'd merged it before

File: package-lock.json
Comment (ValDekh) [reply package-lock.json:1]: Done

**Comment by ValDekh** [comment]
Merged and committed last master brach

**Pull request merged**